### ISSUE TITLE
JsonMappingException while loading dashboard details

### ISSUE THREAD

**Issue by ShridharGoel** [fossasia/open-event-orga-app#1586]
**Describe the bug**
JsonMappingException while loading dashboard details

**To Reproduce**
1. Go to Dashboard.
2. Snackbar appears displaying exception details.

**Expected behavior**
Error should not occur.

**Logs**
```
E/Logger: An exception occurred : com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.Long out of START_OBJECT token
        at [Source: N/A; line: -1, column: -1] (through reference chain: com.eventyay.organizer.data.event.EventStatistics["speakers"])
    java.lang.RuntimeException: com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.Long out of START_OBJECT token
        at [Source: N/A; line: -1, column: -1] (through reference chain: com.eventyay.organizer.data.event.EventStatistics["speakers"])
        at com.github.jasminb.jsonapi.ResourceConverter.readDocument(ResourceConverter.java:236)
        at com.github.jasminb.jsonapi.retrofit.JSONAPIResponseBodyConverter.convert(JSONAPIResponseBodyConverter.java:37)
        at com.github.jasminb.jsonapi.retrofit.JSONAPIResponseBodyConverter.convert(JSONAPIResponseBodyConverter.java:15)
        at retrofit2.ServiceMethod.toResponse(ServiceMethod.java:119)
        at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:218)
        at retrofit2.OkHttpCall.execute(OkHttpCall.java:180)
        at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:41)
        at io.reactivex.Observable.subscribe(Observable.java:11442)
        at retrofit2.adapter.rxjava2.BodyObservable.subscribeActual(BodyObservable.java:34)
        at io.reactivex.Observable.subscribe(Observable.java:11442)
        at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeTask.run(ObservableSubscribeOn.java:96)
        at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:571)
        at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
        at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
        at java.lang.Thread.run(Thread.java:760)
     Caused by: com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.Long out of START_OBJECT token
        at [Source: N/A; line: -1, column: -1] (through reference chain: com.eventyay.organizer.data.event.EventStatistics["speakers"])
        at com.fasterxml.jackson.databind.DeserializationContext.mappingException(DeserializationContext.java:873)
        at com.fasterxml.jackson.databind.deser.std.StdDeserializer._parseLong(StdDeserializer.java:505)
        at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$LongDeserializer.deserialize(NumberDeserializers.java:353)
        at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$LongDeserializer.deserialize(NumberDeserializers.java:331)
        at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:490)
        at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:95)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:260)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:125)
        at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:3761)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2042)
        at com.fasterxml.jackson.databind.ObjectMapper.treeToValue(ObjectMapper.java:2529)
        at com.github.jasminb.jsonapi.ResourceConverter.readObject(ResourceConverter.java:333)
        at com.github.jasminb.jsonapi.ResourceConverter.readDocument(ResourceConverter.java:206)
```

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

**Comment by kush-mish** [comment]
The changes made in the [commit on the open-event-server](https://github.com/fossasia/open-event-server/commit/bdc774a73bfdb491f67a94a53724b2939746b77a) were causing the error.
The preceding commit, #1606 fixes it for the app.

**Comment by ShridharGoel** [comment]
Fixed in #1606
